[2021-12-31 16:06:53,277] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: main_dag.cleanse_spotlight 2021-12-31T16:03:42.129662+00:00 [queued]>
[2021-12-31 16:06:53,303] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: main_dag.cleanse_spotlight 2021-12-31T16:03:42.129662+00:00 [queued]>
[2021-12-31 16:06:53,304] {taskinstance.py:1094} INFO - 
--------------------------------------------------------------------------------
[2021-12-31 16:06:53,305] {taskinstance.py:1095} INFO - Starting attempt 1 of 2
[2021-12-31 16:06:53,305] {taskinstance.py:1096} INFO - 
--------------------------------------------------------------------------------
[2021-12-31 16:06:53,325] {taskinstance.py:1114} INFO - Executing <Task(PythonOperator): cleanse_spotlight> on 2021-12-31T16:03:42.129662+00:00
[2021-12-31 16:06:53,332] {standard_task_runner.py:52} INFO - Started process 31094 to run task
[2021-12-31 16:06:53,336] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'cleanse_spotlight', '2021-12-31T16:03:42.129662+00:00', '--job-id', '184', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpz1ekqi9z', '--error-file', '/tmp/tmph6lds8_j']
[2021-12-31 16:06:53,337] {standard_task_runner.py:77} INFO - Job 184: Subtask cleanse_spotlight
[2021-12-31 16:06:53,380] {logging_mixin.py:109} INFO - Running <TaskInstance: main_dag.cleanse_spotlight 2021-12-31T16:03:42.129662+00:00 [running]> on host 8e99a5d40b8f
[2021-12-31 16:06:53,440] {taskinstance.py:1251} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=main_dag
AIRFLOW_CTX_TASK_ID=cleanse_spotlight
AIRFLOW_CTX_EXECUTION_DATE=2021-12-31T16:03:42.129662+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-31T16:03:42.129662+00:00
[2021-12-31 16:07:05,507] {python.py:151} INFO - Done. Returned value was: None
[2021-12-31 16:07:05,519] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=cleanse_spotlight, execution_date=20211231T160342, start_date=20211231T160653, end_date=20211231T160705
[2021-12-31 16:07:05,573] {local_task_job.py:151} INFO - Task exited with return code 0
